concepts/comparisons/about.md

```diff
@@ -1,2 +1,154 @@
-#TODO: Add about for this concept.
+# About
 
+A comparison operator in python, also called python relational operator, compares the values of two operands and returns `True` or `False` based on whether the condition is met. The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, `"!="`, `"is"`
+
+```python
+>>> 7 > 5
+True
+>>> 99 < 100
+True
+>>> 4 < 4
+False
+>>> 4 <= 4
+True
+>>> 1 >= 1
+True
+>>> 5 == 5
+True
+>>> 6 != 6 # not equal to
+False
+>>> 5 != 6
+True
+```
+
+## Comparison Chaining
+
+Comparisons can be chained arbitrarily, e.g., `x < y <= z` is equivalent to `x < y` `and` `y <= z`, except that `y` is evaluated only once (but in both cases `z` is _not_ evaluated at all when `x < y` is found to be `False`).
+
+Also unlike `C`, expressions like `a < b < c` have the interpretation that is conventional in mathematics.
+
+```python
+>>> x, y, z = 2, 5, 10

```

Comment:
Is unpacking a prereq for this concept?

Reply:
No.  Unpacking appears further "down" the tree.  Left this alone since the example seemed pretty verbose as it stands.  But we probably should change it.

Reply:
Okay. Turned this into simple declaration